fmt/printf.ts

```diff
@@ -916,11 +916,23 @@ class Printf {
 }
 
 /**
- * Converts and format a variable number of `args` as is specified by `format`.
+ * Converts and formats a variable number of `args` as is specified by `format`.
```

Comment:
Mention that users should look at the `printf` module level doc to view what format strings are supported.

@crowlKats we should really have some way to link to module level docs with `@link`

Reply:
@lucacasonato I can look into it on friday

Reply:
Do we want to wait until we can link to the module level doc before this PR lands? I think other than that the PR looks good to me